DNN Platform/Library/Entities/Users/UserController.cs

```diff
@@ -1794,6 +1809,46 @@ public static bool RemoveUser(UserInfo user)
             return retValue;
         }
 
+        /// <summary>
+        /// Delete the contents and folder that belongs to a user in a specific portal
+        /// </summary>
+        /// <param name="user">The user for whom to delete the folder. 
+        /// Note the PortalID is taken to specify which portal to delete the folder from.</param>
+        public static void DeleteUserFolder(UserInfo user)

```

Comment:
For the purposes of future-proofing, can we make this private/internal to avoid adding new public API's (Unless we have to)